contessa/models.py

```diff
@@ -72,19 +73,16 @@ def __table_args__(cls):
         )
 
     def init_row(
-        self,
-        rule: Rule,
-        results: pd.Series,
-        conn: Connector,
-        task_time: datetime = None,
+        self, rule: Rule, results: pd.Series, conn: Connector, context: Dict = None
     ):
         """
         Count metrics we want to measure using pd.Series api and set them to quality check object.
         """
         if results.isnull().any():
             raise ValueError("In results of rule.apply can't be any Null values.")
 
-        self.task_ts = task_time or datetime.now()
+        # todo - add to doc
+        self.task_ts = context.get("task_ts", None) or datetime.now()
```

Comment:
Wouldn't be better to assign some default _task_ts_ sooner in the execution chain (executor or even runner)?

Reply:
good idea, moved to runner